.changeset/great-cougars-buy.md

```diff
@@ -0,0 +1,6 @@
+---
+"github.com/livekit/protocol": major
```

Comment:
oh, this should be `patch` instead of major. The CLI is a bit confusing about this. but manually edititing it here will also work